tests/flow/flow_utils.py

```diff
@@ -0,0 +1,22 @@
+from RLTest import Env
+import multiprocessing as mp
+
+con = None
+
+def run_test_multiproc(env, n_procs, fn, args=tuple()):
```

Comment:
if this is a utility function, I would expect something along the lines of:
```python
#invokes cb on a fork process, passing args to cb
def run_in_fork(cb, args)
```

Reply:
But one of the great advantages of run_test_multiproc is hiding the complexity of creating the connections in the processes so it must also get the env.